Automattic/wp-calypso
# Staging sites: staging sync UI tweaks

Related to https://github.com/Automattic/dotcom-forge/issues/4223 

## Proposed Changes

This pr adds various design tweaks in order to get the staging site card, closer to the designs.
I also notice a bug while we get an error ( it's not working ), so this pr aims to fix that as well.

## Testing Instructions

### Test UI
- Create a WoA site. Add a staging
- Navigate to the hosting configuration page.
- Ensure that the "Synchronize" button is displayed even if the user hasn't selected the sync direction. (It should be disabled).
- After an options is selected, the button must become enabled.
- Ensure that the "Switch to Production Site" button should span the full card width.
- Ensure that ALL checkbox labels are `14px`.
- Ensure that ALL subtitles size is set to `14px`, `italic` and has color: `var(--color-text-subtle) (#646970)`.
- Ensure that ALL checkboxes and their associated labels have a gap of `8px`
- Ensure that the "Danger Zone" box has `16px` padding on each side of it's content (it should be more than that).

### Test Error
- Add [after this line ](https://github.com/Automattic/wp-calypso/blob/trunk/client/state/data-layer/wpcom/sites/sync/status/index.js#L28)the following:
```javascript
if ( status === SiteSyncStatus.COMPLETED ) {
	return dispatch( siteSyncStatusFetchingFailure( {
		siteId: siteId,
		error: 'test'
	} ) );
}
```
- Create a WoA site. Add a staging
- Navigate to the hosting configuration page.
- Initiate a sync.
- When finished you will see an error.
- Press the "Retry" button
- Ensure that you see a request to the sync endpoint and that the status bar appears.

NOTE: I didn't update the label wording etc, as there is a discussion in place. I also agree that the new suggestions are not as descriptive as the ones we have right now.



## Pre-merge Checklist
